backend/server/services/HelloService/index.js

```diff
@@ -0,0 +1,15 @@
+const admin = require('firebase-admin');
+
+const handle = (req, res) => {
+  console.log('Recieved a hello, sending one back');
```

Comment:
Again if this is just a demo I think we should push it to a different branch (or not push it). Can you only push the skeleton to dev and add services later?